src/main/java/org/sagebionetworks/web/client/presenter/TrashPresenter.java

```diff
@@ -14,22 +13,17 @@ public class TrashPresenter
 
   private Trash place;
   private TrashView view;
-  private SynapseReactClientFullContextPropsProvider propsProvider;
```

Comment:
Many of the changes in this PR look like this, where we remove the injected `propsProvider`, which is now unnecessary. Core changes are called out with comments.